dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -242,6 +244,30 @@ public List<URIStatus> listStatus(AlluxioURI path, ListStatusPOptions options)
   public FileOutStream createFile(AlluxioURI path, CreateFilePOptions options)
```

Comment:
As we discussed with @apc999 offline, we should not use proto types in the public api interface, but I know we're using this pattern a lot in the exising code,   we might need a separate PR to refine

Reply:
OK.